meet-data/raw/raw-probe

```diff
@@ -4,41 +4,30 @@
 # Probes for meet results from the 100%RAW Federation (RAW).
 # RAW posts meets to a page separated by year.
 
-
 from bs4 import BeautifulSoup
 import datetime
 import os
 import shutil
 import sys
+sys.path.append('../')
```

Comment:
Honestly, I'm not sure how best to do this in Python. I've just been using symlinks all over the place.

Could you research and find out what best practices are here? I bet they involve using a virtualenv... but I'd like to avoid that if possible.